### PR TITLE
Filter cherrypy access logs.

### PR DESCRIPTION
**Pull request by manthey**
Only log 1 out of 250 tile messages.

Also, switch to the new logprint function for print statements.

Resolves issue #66.

### CODE REVIEW

File: server/rest/tiles.py [hunk 1]
```diff
@@ -51,6 +51,8 @@ def __init__(self, apiRoot):
         apiRoot.item.route('GET', ('test', 'tiles'), self.getTestTilesInfo)
         apiRoot.item.route('GET', ('test', 'tiles', 'zxy', ':z', ':x', ':y'),
                            self.getTestTile)
+        filter_logging.addLoggingFilter(
+            'GET (/[^/ ?#]+)*/item/[^/ ?#]+/tiles/zxy(/[^/ ?#]+){3}', 250)
```
Comment (zachmullen) [inline server/rest/tiles.py]: Do these regexes get compiled on each request, or just once?

Comment (manthey) [inline server/rest/tiles.py]: Just once.

Comment (zachmullen) [inline server/rest/tiles.py]: Great. My only suggestion would be to add a kwarg for 250 so it's obvious what it means.

**Comment by zachmullen** [comment]
LGTM

**Pull request merged**